### PR TITLE
rotating logs

### PR DESCRIPTION
**Pull request by ozgunozerk**
@i1i1 , I had to tinker with `BunyanFomatter` code, I checked and it is working as I expected, but double checking on everything related to logging is alright on Linux side would be helpful :)

Since tokio-tracing team are not responsive to the PR anymore (don't know why), I will probably maintain this fork

### CODE REVIEW

File: src-tauri/src/main.rs [hunk 1]
```diff
@@ -21,13 +21,18 @@ use tracing_subscriber::prelude::*;
 use tracing_subscriber::Layer;
 use tracing_subscriber::{fmt, fmt::format::FmtSpan, EnvFilter};
 
+const KEEP_LAST_N_DAYS: usize = 5;
```
Comment (i1i1) [inline src-tauri/src/main.rs:24]: Do we need 5 days though? I think keeping just one day would be more than enough. You can increase level of logs though and keep smaller time span

Comment (nazar-pc) [reply src-tauri/src/main.rs:24]: I'd rather say 14 days would be preferred, depending on how big the log is

Comment (ozgunozerk) [reply src-tauri/src/main.rs:24]: i recall the daily logs wont reach GBs. It might be around 10MB maybe, but i'm not 100% sure.

Comment (ozgunozerk) [reply src-tauri/src/main.rs:24]: I've done some testing, approx 1MB per hour, so 24MB per day. 
What is our final decision on this parameter in this case?

**Pull request closed without merging**